Security: Add disabled option for cookie samesite attribute (#21472)
grafana/grafana: docs/sources/installation/configuration.md

### cookie_samesite

> Sets the `SameSite` cookie attribute and prevents the browser from sending this cookie along with cross-site requests. The main goal is mitigate the risk of cross-origin information leakage. It also provides some protection against cross-site request forgery attacks (CSRF),  [read more here](https://www.owasp.org/index.php/SameSite). Valid values are `lax`, `strict`, `none` and `disabled`. Default is `lax`. Using value `disabled` will not add any `SameSite` attribute to cookies.

Reviewer: ```suggestion
Sets the `SameSite` cookie attribute and prevents the browser from sending this cookie along with cross-site requests. The main goal is to mitigate the risk of cross-origin information leakage. This setting also provides some protection against cross-site request forgery attacks (CSRF),  [read more about SameSite here](https://www.owasp.org/index.php/SameSite). Valid values are `lax`, `strict`, `none`, and `disabled`. Default is `lax`. Using value `disabled` does not add any `SameSite` attribute to cookies.
```

### cookie_samesite

Sets the `SameSite` cookie attribute and prevents the browser from sending this cookie along with cross-site requests. The main goal is mitigate the risk of cross-origin information leakage. It also provides some protection against cross-site request forgery attacks (CSRF),  [read more here](https://www.owasp.org/index.php/SameSite). Valid values are `lax`, `strict`, `none` and `disabled`. Default is `lax`. Using value `disabled` will not

> add any `SameSite` attribute to cookies.

Reviewer 2: Nit, but why is there a line break here? Considering the line before is so long.
